### PR TITLE
complete api call and set state with returned data

### PR DESCRIPTION
**Pull request by AndrewPomo**

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -3,25 +3,53 @@ import React from 'react';
 import Header from './Header';
 import Overview from './Overview';
 import Review from './Review';
+import queryString from 'query-string';
 
 class App extends React.Component {
-  constructor(props) {
-    super(props);
+  constructor() {
+    super();
```
Comment (ykent4) [inline client/src/components/App.jsx]: Agreed that this is the top component for now and no props is passed in. However I think the proxy server may pass in data like the id of the home to be rendered, so may be safer to just leave in props.

File: server/server.js [hunk 2]
```diff
@@ -14,12 +14,18 @@ app.use(bodyParser.urlencoded({ extended: false }));
...
+      return db.any('SELECT * from listings where id = $1', [req.params.tagId]);
+    })
+    .then((data) => {
+      listingData.overview = data;
+    })
+    .then(() => {
+      res.status(200).send(listingData);
     })
     .catch((error) => {
       res.status(200).send({
```
Comment (ykent4) [inline server/server.js:31]: consider putting in an error code here rather than 200

**Review by ykent4 — COMMENTED** [review]
Just two small comments!

File: server/server.js (hunk 2 above)
Comment (AndrewPomo) [reply server/server.js:31]: I kinda like this setup for development, but it is definitely bad practice for production. I'll fix it later.

**Pull request merged**